graph/types/enums.graphqls

```diff
@@ -50,4 +50,9 @@ enum InviteRequestStatus {
     REJECTED,
     PENDING,
     CANCELLED
+}
+
+enum DiscussionJoinabilitySetting {
+    ALLOW_TWITTER_FOLLOWED,
```

Comment:
should we use twitter nomenclature (followers and friends)?

Reply:
That's a good callout. I will change this!